Home mobile

jeffreyzarate opened the pull request.

jeffreyzarate committed:
Jeffrey Zarate 2:38PM 6/5/2019

Made shared components reponsive. Made tables responsive. Initial
design.

jeffreyzarate committed:
fixed all projects mobile view

jeffreyzarate committed:
Jeffrey Zarate 6/7 1:27PM

Completed mobile version for home page.
Auto scaling navbar.
Auto collapsing table.
Auto scaling logo and buttons.

BrandonSlade reviewed (approved):
Everything looks fine.

mgoshorn commented on src/app/project/view-projects-logic/view-projects-logic.component.scss:
```diff
@@ -145,3 +145,115 @@
+    display: none;
+  }
+
+  // tbody {
```
Remove commented out rules

mgoshorn commented on src/app/project/view-projects-logic/view-projects-logic.component.scss:
```diff
@@ -145,3 +145,115 @@
+    box-shadow: none;
+  }
+
+  #project-image
```
Keep formatting of rules consistent

mgoshorn commented on src/app/project/view-users-projects/view-users-projects.component.scss:
```diff
@@ -126,4 +126,116 @@
+      display: none;
+    }
+  
+    // tbody {
```
Remove commented out rules

mgoshorn reviewed (changes requested):
@jeffreyzarate See above notes

jeffreyzarate committed:
Jeffrey Zarate 6/7 3:40PM

Removed commented code
Made styling on collapsed tables consistent
Fixed logo size for medium-sized screens

jeffreyzarate committed:
Merge branch '190408-dev' into HomeMobile
